POUs/Motion/FB_Axis.TcPOU

```diff
@@ -479,27 +476,45 @@ IF bEnableTemperatureDisable THEN
             END_IF
         END_IF
 
-        //Reset bError to display Error message every 10 seconds if error is still there
+        //Reset bError to display Error message every 30 seconds if error is still there
         IF fbErrorTON.Q THEN
             bError := FALSE;
         END_IF
 
         mControl_TemperatureDisable := TRUE;
         RETURN;
+    ELSE
+        _stAxis.stError.bTempSensorError := FALSE;
     END_IF
 
     //Check scaled tempearture value agianst max. allowed temperature
-    IF fTemperatureValue < _stAXis.stConfig.fMaxTemperature THEN
+    IF fTemperatureValue < _stAXis.stConfig.fMaxTemperature AND NOT bAxisWasOverheated THEN
         bError := FALSE;
         sErrorMessage := '';
         _stAxis.stStatus.fActTemperature := fTemperatureValue;
         _stAxis.stStatus.bHighTempAxisDisabled := FALSE;
         mControl_TemperatureDisable := FALSE;
         RETURN;
+    ELSIF fTemperatureValue < _stAXis.stConfig.fMaxTemperature AND bAxisWasOverheated THEN
```

Comment:
Do we need bAxisWasOverheated here ?
Or is this information already part of 
_stAxis.stStatus.bHighTempAxisDisabled ?